float32 lf_v
float32 lf_p
float32 lb_v
float32 lb_p
float32 rb_v
float32 rb_p
float32 rf_v
float32 rf_p